README.md

```diff
@@ -377,6 +377,12 @@ Parameter | Description | Default
 `tolerations` | Pod's tolerations | `[]`
 `replicaCount.frontend` | Number of frontend pods in your Deployment. [Learn more.](#high-availability) | 2
 `replicaCount.backend` | Number of backend pods in your Deployment. [Learn more.](#high-availability) | 2
+ |  | 
+`hostOverrides.webapp` | Overrides the app configuration if the hostname used to access NeoLoad Web Frontend is different than the value of `services.webapp.host`|
```

Comment:
Because `service.webapp` is used to define service and potentially Ingress for the NLW webapp port.
Here we want to map ti to another port on the side-car.

So we define the `service.weebapp-proxy` service.
But it's hard to identify that this new service is the one used for frontend.
Another solution may be to decorate this new service with something like `service.weebapp-proxy.aliasFor: webapp`.